ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -848,3 +848,81 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: "12h"
+  labels:
+    job-env: aws
+    job-test: e2e
+    job-release: '4.2'
+  name: release-openshift-origin-installer-e2e-aws-4.2-cnv
```

Comment:
This is going to get rolled up in our build-cop stats.  Until we know it's working/stable, i'd rather not have that happen.  Can we start it with a different name (prefix it with "debug-" would be my suggestion, that will keep it out of our queries)

Reply:
@stevekuznetsov has suggested "canary-" as the canonical prefix for this sort of thing.

Reply:
Done.